hello/src/hello_backend_plugin.c

```diff
@@ -54,8 +54,16 @@
 /* These include signatures for plugin and transaction callbacks. */
 #include <clixon/clixon_backend.h> 
 
-static bool hello_state = false;
+/*
+ * This file reflects the state of hello world.  If it exists, hello
```

Comment:
I am a little unsure of the purpose for the file. Does it represent "system state" not present in the config?